skimage/measure/_marching_cubes_lewiner_cy.pyx

```diff
@@ -44,8 +44,8 @@ def remove_degenerate_faces(vertices, faces, *arrays):
     vertices_map1 = vertices_map0.copy()
     faces_ok = np.ones(len(faces), dtype=np.int32)
 
-    cdef float [:, :] vertices_ = vertices
-    cdef float [:] v1, v2, v3
+    cdef cnp.float32_t [:, :] vertices_ = vertices
+    cdef cnp.float32_t [:] v1, v2, v3
```

Comment:
Here `numpy` is `cimport`ed as `np`... Maybe should we change it to `cnp` for consistency?
```suggestion
    cdef np.float32_t [:, :] vertices_ = vertices
    cdef np.float32_t [:] v1, v2, v3
```